Backend/app.py

```diff
@@ -52,67 +54,99 @@ def getDatabaseClient():
   on if we are in Dev or Prod
   '''
   if prod == 'True':
+    logger.info("Using AWS Prod Database")
     return boto3.resource('dynamodb')
   else:
-    print("Returning the Dev resource")
+    logger.info("Using Local Dev Database")
     return boto3.resource('dynamodb', endpoint_url='http://localhost:8000')
 
 @auth.verify_password
 def verify_password(email_or_token, password):
+  logger.info(str(request))
   dynamoDBInstance = getDatabaseClient()
-  return MentiiAuth.verify_password(email_or_token, password, dynamoDBInstance, appSecret)
+  response = MentiiAuth.verify_password(email_or_token, password, dynamoDBInstance, appSecret)
+  logger.info(str(response))
+  return response
 
 @app.route('/', methods=['GET', 'POST'])
 def index():
-  return cr.createEmptyResponse(200)
+  logger.info(str(request))
+  response = ResponseCreation.createEmptyResponse(200)
+  logger.info(str(response))
+  return response
 
 @app.route('/register/', methods=['POST', 'OPTIONS'])
 def register():
+  logger.info(str(request))
   status = 200
   if request.method =='OPTIONS':
-    return cr.createEmptyResponse(status)
+    flaskResponse = ResponseCreation.createEmptyResponse(status)
+    logger.info(str(flaskResponse))
```

Comment:
This is related for all OPTIONS requests but I don't think logging is needed since its just an informational request made by browsers to learn information about the endpoint